browser/scripts/clone_template_html.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const path = require('path');
+const process = require('process');
+
+const cloneTemplateHTML = () => {
+  const cwd = process.cwd();
+  
+  if (!fs.existsSync(cwd + '/.generated')) {
+    fs.mkdirSync(cwd + '/.generated');
+  }
+
+  if (!fs.existsSync(cwd + '/.generated/index.html')) {
+    fs.copyFileSync(
+      path.resolve(cwd + '/../templates/index.html'),
+      path.resolve(cwd + '/.generated/index.html')
+    );
+  }
+};
+
+cloneTemplateHTML();
```

Comment:
If a .generated directory does not exist and/or a .generated/index.html file does not exist, this script will create them by copying our template.  This is simply to make things easier for parcel right now.  I think this will have to be a bit more robust later, but as an unblocking step it serves its purpose.